src/Tests/InstrEngineTests/InstrEngineTests/EmbeddedResources/InvalidCSharp/readme.txt

```diff
@@ -0,0 +1,6 @@
+﻿-   ByRefLikeInvalidCSharp.il
+    Original source code from the dotnet runtime GitHub repo (https://github.com/dotnet/runtime/blob/4881a639e7c3f27b5a8d2d160e234d8055333cda/src/tests/Loader/classloader/generics/ByRefLike/InvalidCSharp.il)
+-   RefInvalidCSharp.il
+    Original source code from the dotnet runtime GitHub repo (https://github.com/dotnet/runtime/blob/053a472be0143bda0891c715d7b2ce7398538127/src/tests/Loader/classloader/RefFields/InvalidCSharp.il)
+-   RefStructsTests.cs
```

Comment:
>RefStructsTests.cs [](http://example.com/codeflow?start=4&length=18)

This link is broken. Also, the license in the file indicates that it is from Microsoft, not the dotnet foundation.

Reply:
Which link do you think is broken? I just checked both and didn't have issues with either of them.

Reply:
My guess is you might've been getting the comma as part of the link; I switched to using `and` just in case that's an issue.